.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,24 +7,24 @@ assignees: ''
 
 ---
 
-**Describe the bug**
+### Describe the bug
```

Comment:
Why are we using mardown Heading level 3 instead of Heading level 1?

Reply:
The linter wasn't happy about the bold text, and wanted headers. I chose the H3 for readability because 

# Describe the bug

seems a bit hostile compared to 

### Describe the bug

which seems more close to your original intent with the bold text. WDYT?

Reply:
Please continue this thread in https://github.com/qonto/prometheus-rds-exporter/pull/126 if needed.